index.html

```diff
@@ -0,0 +1,531 @@
+              <ul class="header__pc-nav">
+                <li class="header__pc-item">
+                  <a href="" class="link">Home</a>
+                </li>
+                <li class="header__pc-item">
+                  <a href="" class="link">Product</a>
+                </li>
+                <li class="header__pc-item">
+                  <a href="" class="link">Pricing</a>
+                </li>
+                <li class="header__pc-item">
+                  <a href="" class="link">Contact</a>
+                </li>
+              </ul>
+            </li>
+            <li class="header__item">
+              <img src="./assets/search.svg" alt="" class="icon" />
+            </li>
+            <li class="header__item">
+              <img src="./assets/cart.svg" alt="" class="icon" />
+            </li>
+            <li class="header__item header__item--last">
+              <img src="./assets/burger.svg" alt="" class="icon" />
+            </li>
+          </ul>
+        </div>
+        <ul class="header__mobile-nav">
+          <li class="header__mobile-item header__mobile-item--second">Home</li>
+          <li class="header__mobile-item header__mobile-item--second">
+            Product
+          </li>
+          <li class="header__mobile-item header__mobile-item--second">
+            Pricing
+          </li>
+          <li class="header__mobile-item header__mobile-item--second">
+            Contact
+          </li>
+        </ul>
+      </div>
+    </header>
+    <main class="main">
+      <section class="banner">
+        <div class="banner__wrapper">
+          <h1 class="heading heading--xl heading--center">
+            Experts are here solve your business problem.
+          </h1>
+          <p class="text text--center text--max-lg text--secondary">
+            We know how large objects will act, but things on a small scale just
+            do not act that way.
+          </p>
+          <div class="banner__actions">
+            <button class="btn">
+              <span class="btn__text">Get Quote Now</span>
+            </button>
+            <button class="btn btn--secondary">
+              <span class="btn__text btn__text--secondary">Learn More</span>
+            </button>
+          </div>
+        </div>
+      </section>
+      <section class="service">
+        <header class="service__header">
+          <h2 class="heading heading--md heading--center">WHY CHOOSE US</h2>
+          <p class="text text--center text--max text--secondary">
+            Problems trying to resolve the conflict between the two major realms
+            of Classical physics: Newtonian mechanics
+          </p>
+        </header>
+        <ul class="list">
+          <li class="list__item">
+            <div class="service__card">
+              <div class="service__card-header">
+                <img src="./assets/toolbox.svg" alt="" />
+                <h5 class="heading__sm">Digital Marketing</h5>
```

Comment:
This element is out of its block and also it is written as an element but it sounds like a modifier for example here `heading--md` and here you wrote `heading__sm`. you can keep following the same structure